bot/role_management/role_management.py

```diff
@@ -45,7 +45,9 @@ async def toggle_module(self, ctx: commands.Context, *, str_modules: str):
             str_modules (str): A string containing abbreviations of all the modules a user would like to toggle.
         """
         if ctx.channel.id != self.ch_role.id:
-            await ctx.message.delete()
+            if not self._db_connector.is_botonly(ctx.channel.id):
+                await ctx.message.delete()
+
```

Comment:
Remove blank line

Reply:
Nope. I want to visually separate the code in the if block from the rest.